src/Everything.agda

```diff
@@ -5,6 +5,7 @@ module Everything where
 open import Cat
 open import Cat.Cartesian
 open import Functor
+open import Functor.Cat
```

Comment:
Do we really want to have to keep Everything.agda up to date?  Seems like a good way to introduce unnecessary merge conflicts.  BTW nice work upgrading my script to a makefile!  :)

Reply:
We could remove it and add it to `.gitignore`

Reply:
Cool I'll do that in a followup